ui/src/main/js/page/stocks/index.js

```diff
@@ -126,8 +127,39 @@ class Stocks extends React.Component {
       });
   }
 
+  updateStock(message) {
+    let tmpStocks = this.state.stocks;
+    let updatedStock = JSON.parse(message.body);
+//    tmpStocks.push(updatedStock);
+//    tmpStocks.sort(function(a, b) {
+//        return b.entity.percentageChange - a.entity.percentageChange;
+//    });
+//    tmpStocks.slice(0, this.state.pageSize);
+    let stock = tmpStocks.find((x) => x.entity.symbol===updatedStock.symbol);
+    if(typeof stock !== "undefined") {
+        updatedStock._changeEvent = "increase";
+        if (parseInt(updatedStock.price) > parseInt(stock.price)) {
+            updatedStock._changeEvent = 'increase';
+
+        } else if (parseInt(updatedStock.price) < parseInt(stock.price)) {
+            updatedStock._changeEvent = 'decrease';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/modernmaster/forecaster/pullRequest?prid=7290232)
```suggestion
            updatedStock._changeEvent = "decrease";
```